spaceship.zsh

```diff
@@ -587,6 +595,44 @@ spaceship_ruby() {
     "$SPACESHIP_RUBY_SUFFIX"
 }
 
+# ELIXIR
+# Show current version of Elixir
+spaceship_elixir() {
+  [[ $SPACESHIP_ELIXIR_SHOW == false ]] && return
+
+  # Show versions only for Ruby-specific folders
+  [[ -f mix.exs || -n *.ex(#qN) || -n *.exs(#qN) ]] || return
+
+  local elixir_version
+
+  if _exists kiex; then
+    elixir_version="${ELIXIR_VERSION}"
+  elif _exists exenv; then
+    elixir_version=$(exenv version-name)
+  elif _exists elixir; then
+    elixir_version=$(elixir -v | grep "Elixir" --color=never | cut -d ' ' -f 2)
+  else
+    return
+  fi
+
+  if [[ "${elixir_version}" == "system" ]]; then
+    if _exists elixir; then
+      elixir_version=$(elixir -v | grep "Elixir" --color=never | cut -d ' ' -f 2)
```

Comment:
I included the SPACESHIP_ELIXIR_DEFAULT_VERSION code and removed the duplicate line